astropy/table/table.py

```diff
@@ -239,7 +239,9 @@ class TableColumns(OrderedDict):
         Column objects as data structure that can init dict (see above)
     """
 
-    def __init__(self, cols={}):
+    def __init__(self, cols=None):
+        if cols is None:
+            cols = {}
```

Comment:
```suggestion
        cols = {} if cols is None else cols
```
I would suggest something like
```python
        cols = cols or {}
```
but things like empty lists, empty tuples, or other "falsy" objects would  get remapped to `{}` and I don't know the consequences of that.